UnitTests/project.json

```diff
@@ -26,10 +26,11 @@
           }
         }
     },
-    "dependencies": {
-        "jose-jwt": "2.0.0",
-        "xunit": "2.1.0"
-    },
+  "dependencies": {
+    "jose-jwt": "2.0.0",
+    "Microsoft.NETCore.App": "1.1.0",
```

Comment:
mm...understood it is tests, but why do we need this reference to Microsofr.NETCore.App ?